### PR TITLE
feat 配置平台-业务主机 右侧过滤，增加sn批量查询功能，方便根据sn批量查询主机

### PR DESCRIPTION
**Pull request by caotao**
### 新加的功能:
- 蓝鲸配置平台-业务主机 右侧过滤，增加sn批量查询功能，方便根据sn批量查询主机

### CODE REVIEW

File: src/scene_server/host_server/logics/hostsearch.go [hunk 1]
```diff
@@ -690,7 +691,10 @@ func (sh *searchHost) searchByHostConds() error {
 		Sort:      sh.hostSearchParam.Page.Sort,
 	}
 
+	blog.Infof("3 searchByHostConds query:%v", query)
```
Comment (wusendong) [inline src/scene_server/host_server/logics/hostsearch.go]: 请问这个3是什么意思？

Comment (caotao) [reply src/scene_server/host_server/logics/hostsearch.go]: 这里优化下，当时log标识，调试用的

File: src/common/paraparse/host.go [hunk 2]
```diff
@@ -154,3 +154,20 @@ func ParseHostIPParams(ipCond metadata.IPInfo, output map[string]interface{}) er
...
+//SN 批量查询
+func ParseSnParam(sn []string, output map[string]interface{}) error {
+	if len(sn) < 1 {
+		return nil
+	}
+	c := make(map[string]interface{})
+	c[common.BKDBIN] = sn
+
+	snCon := make(map[string]map[string]interface{})
+	snCon["bk_sn"] = c
```
Comment (rentiansheng) [inline src/common/paraparse/host.go]: bk_sn 定义为常量，是否比较。

File: src/common/paraparse/host.go [hunk 3]
```diff
@@ -154,3 +154,20 @@ func ParseHostIPParams(ipCond metadata.IPInfo, output map[string]interface{}) er
...
+
+	orCond := make([]map[string]map[string]interface{}, 0)
+	output[common.BKDBAND] = append(orCond, snCon)
```
Comment (rentiansheng) [inline src/common/paraparse/host.go]: 这里是BKDBAND, 上面定义的变量是orCond是不是有冲突了

**Comment by rentiansheng** [comment]
@breezelxp 这个是不是同时需要cherry pick 到v3.4.x中

**Comment by breezelxp** [comment]
> @breezelxp 这个是不是同时需要cherry pick 到v3.4.x中
是的

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-cmdb?pullRequest=2457) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-cmdb?pullRequest=2457) before we can accept your contribution.<br/><hr/>**ericcao** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-cmdb?pullRequest=2457) it.</sub>

File: src/common/paraparse/host.go (hunk 3 above)
Comment (caotao) [reply src/common/paraparse/host.go]: 不冲突，同+131 ParseHostIPParams 的输出

File: src/common/paraparse/host.go (hunk 2 above)
Comment (caotao) [reply src/common/paraparse/host.go]: 已经处理,tks

**Comment by caotao** [comment]
为通过license/cla 修改了 commit 的 提交email

**Comment by caotao** [comment]
这个pr还有什么问题吗 @breezelxp @wusendong @rentiansheng

**Pull request closed without merging**